core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToTemplatePreparationObjectConverter.java

```diff
@@ -297,8 +297,7 @@ private void decorateBuilderWithPlacement(StackV4Request source, Builder builder
         PlacementSettingsV4Request placementSettings = source.getPlacement();
         if (placementSettings != null) {
             String region = placementSettings.getRegion();
-            String availabilityZone = placementSettings.getAvailabilityZone();
-            builder.withPlacementView(new PlacementView(region, availabilityZone));
+            builder.withPlacementView(new PlacementView(region, region));
```

Comment:
why did you remove the az field?

Reply:
this field is not even used in the template generation. Also we are nor always populated this value

Reply:
If I understand correctly, we wouldn't set the az via API?